lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -3298,6 +3298,11 @@ bool TypeSystemSwiftTypeRef::IsTypedefType(opaque_compiler_type_t type) {
   if (mangled_name == "$sSo18NSNotificationNameaD" ||
       mangled_name == "$sSo9NSDecimalaD")
     return impl();
+
+  // SIMD types have some special handling in the compiler, causing divergences 
+  // on the way SwiftASTContext and TypeSystemSwiftTypeRef view the same type.
+  if (mangled_name.GetStringRef().contains_insensitive("simd"))
```

Comment:
The string "simd" is really short and could appear in many types. Is there a way to use startswith() or at least search for "4simd" to make false positives slightly less likely? Otherwise this LGTM!